api/service_handlers.go

```diff
@@ -175,7 +175,15 @@ func serviceUnbindApp(c echo.Context) error {
 		return err
 	}
 
-	if err = manager.UnbindApp(c.Request().Context(), c.Param("instance")); err != nil {
+	appName, err := formValue(c.Request(), "app-name")
```

Comment:
We should be using `app_name` here to be compliant with other "multi-word" parameters. If you prefer, I think just `app` is even better.